internal/am/job_tracker.go

```diff
@@ -12,6 +12,53 @@ import (
+	"1c2550f1-3fc0-45d8-8bc4-4c06d720283b": {}, // Scan for viruses in directories
+	"56eebd45-5600-4768-a8c2-ec0114555a3d": {}, // Generate transfer structure report
+	"2584b25c-8d98-44b7-beca-2b3ea2ea2505": {}, // Change object and directory filenames
+	"a329d39b-4711-4231-b54e-b5958934dccb": {}, // Change transfer name
+	"2522d680-c7d9-4d06-8b11-a28d8bd8a71f": {}, // Identify file format
+	"1a136608-ae7b-42b4-bf2f-de0e514cfd47": {}, // Load rights
+	"303a65f6-a16f-4a06-807b-cb3425a30201": {}, // Characterize and extract metadata
+	"d51a7ed8-9cf4-424b-9671-85fd8b5b95aa": {}, // Load PREMIS events from metadata/premis.xml
+	"a536828c-be65-4088-80bd-eb511a0a063d": {}, // Validate formats
+	"307edcde-ad10-401c-92c4-652917c993ed": {}, // Generate METS.xml document
+	"675acd22-828d-4949-adc7-1888240f5e3d": {}, // Parse external METS
+	// Ingest jobs.
+	"15a2df8a-7b45-4c11-b6fa-884c9b7e5c67": {}, // Identify manually normalized files
+	"5b0042a2-2244-475c-85d5-41e4b11e65d6": {}, // Validate preservation derivatives
+	"78b7adff-861d-4450-b6dd-3fabe96a849e": {}, // Check for manual normalized files
+	"47dd6ea6-1ee7-4462-8b84-3fc4c1eeeb7f": {}, // Check for submission documentation
+	"2a62f025-83ec-4f23-adb4-11d5da7ad8c2": {}, // Assign checksums and file sizes to submissionDocumentation
+	"11033dbd-e4d4-4dd6-8bcf-48c424e222e3": {}, // Change file and directory names in submission documentation
+	"1ba589db-88d1-48cf-bb1a-a5f9d2b17378": {}, // Scan for viruses in submission documentation
+	"1dce8e21-7263-4cc4-aa59-968d9793b5f2": {}, // Identify file format
+	"33d7ac55-291c-43ae-bb42-f599ef428325": {}, // Characterize and extract metadata on submission documentation
+	"b0ffcd90-eb26-4caf-8fab-58572d205f04": {}, // Process JSON metadata
+	"e4b0c713-988a-4606-82ea-4b565936d9a7": {}, // Move metadata to objects directory
+	"dc9d4991-aefa-4d7e-b7b5-84e3c4336e74": {}, // Assign file UUIDs to metadata
+	"b6b0fe37-aa26-40bd-8be8-d3acebf3ccf8": {}, // Assign checksums and file sizes to metadata
+	"b21018df-f67d-469a-9ceb-ac92ac68654e": {}, // Change file and directory names in metadata
+	"8bc92801-4308-4e3b-885b-1a89fdcd3014": {}, // Scan for viruses in metadata
+	"b2444a6e-c626-4487-9abc-1556dd89a8ae": {}, // Identify file format of metadata files
+	"04493ab2-6cad-400d-8832-06941f121a96": {}, // Characterize and extract metadata on metadata files
+	"873b428f-2c86-42b6-b463-aeda925bf559": {}, // Load persistent identifiers from external file
+	"ccf8ec5c-3a9a-404a-a7e7-8f567d3b36a0": {}, // Generate METS.xml document
+	"523c97cc-b267-4cfb-8209-d99e523bf4b3": {}, // Add README file
+	"3e25bda6-5314-4bb4-aa1e-90900dce887d": {}, // Prepare AIP
+	"d55b42c8-c7c5-4a40-b626-d248d2bd883f": {}, // Compress AIP
+	"3f543585-fa4f-4099-9153-dd6d53572f5c": {}, // Verify AIP
+	"20515483-25ed-4133-b23e-5bb14cab8e22": {}, // Store the AIP
+	"48703fad-dc44-4c8e-8f47-933df3ef6179": {}, // Index AIP
+	"b7cf0d9a-504f-4f4e-9930-befa817d67ff": {}, // Clean up after storing AIP
+}
```

Comment:
I was initially planning to split this map and pass two different maps from the activities (one for the transfer jobs and the other for the ingest jobs), adding a new field to the `JobTracker` struct, setting it on the `NewJobTracker` and checking it while filtering. But in the end I thought that having all the jobs here may be better. Happy to change it back if you think that's better.